pkg/kube/mounts/manager.go

```diff
@@ -76,21 +76,21 @@ func (mm *MountManager) removeVolume() error {
 		}
 	}
 
-	return kubereflect.AssignValueTo(mm.Target, VolumesFieldPath, volumes)
+	return kubereflect.AssignValueTo(mm.Target, VolumesFieldPath, desiredVolumes)
 }
 
 func (mm *MountManager) removeVolumeMounts() error {
```

Comment:
in this function we don't log anything? seems to be an important function which should log something

Reply:
@waseem-h is adding logs in #3 to cover this.